### PR TITLE
Family stream prompts

### PR DESCRIPTION
**Pull request by cjmaxik**
We need 16 prompts for each participant.

### CODE REVIEW

**Comment by cjmaxik** [comment]
@tioopental need your help to fill the rest of the prompts

**Comment by tioopental** [comment]
Should be fine

File: src/prompts/family/neuro.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    'aw',
+    'hm',
+    'meow',
+    'filtered',
+    'KEKWa',
+    'misgenders someone',
+    'trauma dump',
+    'calls someone an AI',
+    'tries to leave',
+    'donowalls',
```
Comment (cjmaxik) [inline src/prompts/family/neuro.js:16]: Donowalls whom exactly? I doubt there will be chat interactions from Neuro, just like in previous collabs. Please reconsider.

File: src/prompts/family/neuro.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  prompts: [
+    'aw',
+    'hm',
+    'meow',
+    'filtered',
+    'KEKWa',
+    'misgenders someone',
+    'trauma dump',
+    'calls someone an AI',
```
Comment (cjmaxik) [inline src/prompts/family/neuro.js:14]:
```suggestion
    'calls one of them an AI',
```
Prevent orphan text.

File: src/prompts/family/neuro.js (hunk 1 above)
Comment (cjmaxik) [reply src/prompts/family/neuro.js:16]: Alright, we'll have it for now, unless we end up thinking about something more suitable.

**Pull request merged**